app/locales/en/translations.js

```diff
@@ -1255,6 +1255,9 @@ export default {
         date: 'Date',
         author: 'Author'
       },
+      visitType: 'Visit Type',
```

Comment:
I must have missed this on the original PR.  These localization values are not organized properly.  They appear to be labels, so they should have a labels prefix, not "form".  Not sure what "form" refers to but the localizations in this section need to refactored.